### DISCUSSION TITLE
install/commercial/port-requirements

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4244] [Announcements]
# install/commercial/port-requirements

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/commercial/port-requirements.html

**Comment by san3Xian** [comment]
2381 TCP 用于 etcd metric port，也需要打开

**Comment by ModetaNiu** [comment]
fixed,  see https://github.com/DaoCloud/DaoCloud-docs/pull/4546
